custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -54,6 +55,22 @@ def __init__(self, runtime):
         self.verbose = runtime.options.verbose
         self.run_mode = RunMode(runtime.options.subparser_name)
         self.raw_arguments = sys.argv
+        self.target_type = None
+
+        if self.run_mode != RunMode.NEW:
+            # if target type and class labels are not provided assume it is regression
+            if self.options.target_type is None:
+                if self.options.negative_class_label is None:
+                    self.options.target_type = TargetType.REGRESSION.value
+                elif None not in [
+                    self.options.positive_class_label,
+                    self.options.negative_class_label,
+                ]:
+                    self.options.target_type = TargetType.BINARY.value
+                else:
+                    raise DrumCommonException("Target type can not be resolved")
```

Comment:
Make the error more informative.. like saying no postive or negative classes were provided and no --target type ...

Reply:
Eventually, I remove this exception as it doesn't make sense: currently, if labels are not provided, it is regression.

I'll check a bit more if it is already better to make --target-type required